rudder-core/src/main/scala/com/normation/rudder/services/quicksearch/QuickSearchBackendImpl.scala

```diff
@@ -276,6 +276,12 @@ object QSLdapBackend {
    * Mapping between attributes and their filter in the backend
    */
   object QSAttributeLdapFilter {
+
+    //the list of value to avoid in OC for matching
+    val ocRudderTypes = Set("top", OC_RUDDER_NODE, OC_POLICY_SERVER_NODE, OC_NODE
+                            , OC_RULE, OC_PARAMETER, OC_RUDDER_NODE_GROUP)
```

Comment:
We only select object in OC_NODE, OC_RUDDER_NODE, OC_RULES, OC_PARAMETER, OC_RUDDER_GROUP (and their children), so we don't need to filter these one.